src/aui-layout/js/aui-layout-builder-remove-row.js

```diff
@@ -147,6 +149,26 @@ LayoutBuilderRemoveRow.prototype = {
     },
 
     /**
+     * Protected method to set enableRemoveRows atribute as false.
+     *
+     * @method _disableRemoveRows
+     * @protected
+     */
+    _disableRemoveRows: function() {
+        this.set('enableRemoveRows', false);
+    },
+
+    /**
+     * Protected method to set enableRemoveRows atribute as true.
+     *
+     * @method _enableRemoveRows
+     * @protected
+     */
+    _enableRemoveRows: function() {
```

Comment:
This can cause problems if the user had called `LayoutBuilder`  with `enableRemoveRows` as false to begin with. Maybe store the previous value when `_disableRemoveRows` is called so that we won't reenable it here by mistake?